src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -35,104 +34,85 @@ const DEFAULT_REASONS = {
 
 class CloseOrderModal extends Component {
   static propTypes = {
-    orderId: PropTypes.string,
-    workflowStatus: PropTypes.string,
+    orderNumber: PropTypes.string,
     closingReasons: PropTypes.arrayOf(PropTypes.object),
     closeOrderSubmit: PropTypes.func.isRequired,
+    isOpen: PropTypes.bool.isRequired,
```

Comment:
I still don't see any reason to use this prop. You control to render this model or not in parent component